jvb/rootfs/etc/services.d/jvb/run

```diff
@@ -2,9 +2,14 @@
 
 JAVA_SYS_PROPS="-Dnet.java.sip.communicator.SC_HOME_DIR_LOCATION=/ -Dnet.java.sip.communicator.SC_HOME_DIR_NAME=config -Djava.util.logging.config.file=/config/logging.properties"
 
-if [[ ! -z "$DOCKER_HOST_ADDRESS" ]]; then
-    LOCAL_ADDRESS=$(hostname -I | cut -d " " -f1)
-    JAVA_SYS_PROPS="$JAVA_SYS_PROPS -Dorg.ice4j.ice.harvest.NAT_HARVESTER_LOCAL_ADDRESS=$LOCAL_ADDRESS -Dorg.ice4j.ice.harvest.NAT_HARVESTER_PUBLIC_ADDRESS=$DOCKER_HOST_ADDRESS"
+PUBLIC_ADDRESS="$(curl -4qs --connect-timeout 15 icanhazip.com)"
```

Comment:
Because you've removed the `DOCKER_HOST_ADDRESS` it will not be possible to run the container without an internet connection in a LAN environment anymore I think. Additionally the docs for this env var are still there.